notebooks/alerts_intro_eda.ipynb

```diff
@@ -0,0 +1,651 @@
+{
```

Comment:
<p>The last two outputs are the same, you should change the last row to:</p><p> </p><p><code>print(f'Least recent data point is from timestamp {df["timestamp"].min()}')</code></p><p> </p><p><code>min</code>  instead of <code>max</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-alert-prediction-public/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>